proxy/src/metrics.rs

```diff
@@ -277,15 +277,16 @@ pub struct ComputeConnectionLatencyGroup {
     protocol: Protocol,
     cold_start_info: ColdStartInfo,
     outcome: ConnectOutcome,
-    excluded: LatencyExclusions,
+    component: LatencyComponents,
 }
 
 #[derive(FixedCardinalityLabel, Copy, Clone)]
-pub enum LatencyExclusions {
+pub enum LatencyComponents {
```

Comment:
Oh, I thought about it, but, there is no e2e latency anymore (correct me if I am wrong) and I think, it's useful to have it.

Also, with that granularity, how can we measure p90, p99 etc.? I don't think that p90 of Proxy + p90 of Compute is the same as p90 of Proxy and Compute.

Reply:
Let me deploy it to dev-us-east-2 and see if summing over the components gives similar values to what we already observer